EXAMPLES.md

```diff
@@ -43,3 +43,5 @@
 | [146 Parks](https://146parks.blog/)                                | https://github.com/scouten/146parks.blog                 |
 | [films.mlcdf.fr](https://films.mlcdf.fr)                           | https://github.com/mlcdf/films                           |
 | [Mish Ushakov](https://mish.co)                                    |                                                          |
+| [castor](https://castorisdead.xyz)                                 | https://github.com/whoisYoges/website                    |
+| [pwnwriter](https://pwnwriter.xyz)                                 | https://github.com/pwnwriter/www
```

Comment:
This gives a 404?

Reply:
This site is under dev... will be updated within one or two day.

Reply:
Hi, i've removed the 404 site and add new websites built with zola. Hope the pr will get accepted:)